tests/api-mongo.test.js

```diff
@@ -259,3 +328,23 @@ describe('MongoDB Words', () => {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  describe('/undefinedRoute', () => {
+    it('should return response status of 404', (done) => {
+      chai.request(server).get('/undefined').end((_, res) => {
```

Comment:
You should make a new command in the `commands.js` file and then import that new function into this test file so that we can exclude server request logic.